temoa/extensions/breakeven/breakeven.py

```
@@ -253,11 +253,11 @@ def sensitivity(dat, techs):
                     urc,
                     scal_CAP[t, v],
-                    scal_CAP[t, v] * instance.CostInvest[t, v],
-                    scal_CAP[t, v] * instance.CostFixed[v, t, v],  # Use the FC of the first period
-                    instance.CostInvest[t, v],
-                    instance.CostFixed[v, t, v],
-                    value(instance.V_Capacity[t, v]),
+                    scal_CAP[t, v] * instance.cost_invest[t, v],
+                    scal_CAP[t, v] * instance.cost_fixed[v, t, v],  # Use the FC of the first period
+                    instance.cost_invest[t, v],
+                    instance.cost_fixed[v, t, v],
+                    value(instance.v_capacity[t, v]),
```

Ruff (0.14.3)

`value` may be undefined, or defined from star imports

(F405)